server/shared/publishing/helpers.js

```diff
@@ -71,12 +71,12 @@ function getRepositoryCatalog() {
   });
 }
 
-function updateRepositoryCatalog(repository, publishedAt) {
+function updateRepositoryCatalog(repository, publishedAt, updateInfo = true) {
```

Comment:
Nitpick: Would prefer if this was an `options` object as it already has two arguments supplied. But can leave it as is for now.

Reply:
I wanted to do that too, but since function is used in multiple places, didn't want to mess up existing code as part of this fix.